components/omega/doc/design/Eos.md

```diff
@@ -0,0 +1,196 @@
+(omega-design-eos)=
+
```

Comment:
We should leave the design doc out of this PR, since it's in a separate PR and will be merged before this one.

Reply:
OK, can you check the differences? I made minor changes to address Xylar's comments.